content/_includes/server-config.md

```diff
@@ -63,12 +63,12 @@ tls:
 # The API configuration. The APIs exposed by the KES server can
 # be adjusted here. Each API is identified by its API path.
 #
-# In general, the KES server uses reasonable defaults for all APIs.
+# In general, the KES server uses sane defaults for all APIs.
 # Only customize the APIs if there is a real need.
 # 
 # Disabling authentication for an API must be carefully evaluated.
-# One example when disabling authentication may be justified could
-# be the readiness probe in a Kubernetes environment.
+# One example, when disabling authentication may be justified, would
+# be the liveness and readiness probes in a Kubernetes environment.
```

Comment:
maybe
```suggestion
# Possible examples are the liveness and
# readiness probes in a Kubernetes environment.
```

Reply:
Suggest we make this change in the KES repo.